### PR TITLE
Checkpoint 2) User Sign Up

### PR DESCRIPTION
**Pull request by john-pickett**
New users can sign up and receive an email confirmation. They can also edit their profile and logout.

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (cbeck) [inline spec/models/user_spec.rb:1]: You can remove this. Devise is very well tested.

**Review by cbeck — APPROVED** [review]
Looks good! You can remove that user spec - devise is very well tested.

**Comment by cbeck** [comment]
👍 Also - I don't want to be the bottleneck, so feel free to approve your own PRs if you are continuing work that day.

**Comment by john-pickett** [comment]
Great, thanks. I'll keep that in mind as I work through the rest of the project.

**Pull request merged**